src/ussd_clinic_rapidpro.js

```diff
@@ -0,0 +1,27 @@
+go.app = function() {
+    var vumigo = require("vumigo_v02");
+    var App = vumigo.App;
+    var EndState = vumigo.states.EndState;
+
+    var GoNDOH = App.extend(function(self) {
+        App.call(self, "state_start");
+        var $ = self.$;
+
+        self.init = function() {
+        };
+
+        self.states.add("state_start", function(name) {
+            return new EndState(name, {
+                text: $(
+                    'Hello mom! You can reply to any MomConnect message with a question, compliment or complaint and ' +
+                    'our team of experts will get back to you.'
```

Comment:
I don't think we should be using the same text here as for the basic setup of the public line, otherwise we can't be sure that things are setup correctly. I think we  should rather use the text in the first screen of the clinic line here.